test/entry/entry-single-arg.test.js

```diff
@@ -10,7 +10,9 @@ describe('single entry flag', () => {
         const { stdout, stderr } = run(__dirname);
         const summary = extractSummary(stdout);
         const outputDir = 'entry/bin';
-        expect(summary['Output Directory']).toContain(outputDir);
+        const outDirectoryFromCompiler = summary['Output Directory'].split(sep);
+        const outDirToMatch = outDirectoryFromCompiler.slice(outDirectoryFromCompiler.length - 2, outDirectoryFromCompiler.length).join('/');
```

Comment:
`.join('/')` is not correct. Windows don't have `/` in their directory paths

Reply:
I just used `'/'` to make it in the format of the `outputDir` i.e `entry/bin` .

Reply:
Yes, I know. Webpack under the hoods takes care of it. And when it's parsed, it's transformed with the path convention used by the operative system.

The use `path` utility to deal with paths.